### DISCUSSION TITLE
How to connect tableu to R

### DISCUSSION THREAD

**Discussion by AnthonySSSSS** [teamalgoritma/community#437] [Q&A] [answered]
![Screenshot_20221104-025837_WhatsApp](https://user-images.githubusercontent.com/112844100/199821914-58ccd5ab-ead1-43da-8b79-234e2110a051.jpg)
help me connect tableu to R for covid data processing, and visualization to shinyapps. Thank you

**Comment by kevbow** [comment, marked as answer]
Dear @AnthonySSSSS,

Terima kasih atas pertanyaan yang disampaikan, pertanyaan Anda sudah kami terima dan akan kami carikan referensi untuk hal ini. 
Terima kasih.

**Reply by kevbow** [reply]
Halo, berikut saya berikan referensi untuk melakukan hal ini ya.
Dengan catatan bahwa data yang digunakan di tableau adalah data statis bertipe .csv dan kode R digunakan untuk melakukan komputasi tambahan pada Tableau.
- https://help.tableau.com/current/pro/desktop/en-us/r_connection_manage.htm#rserve
- https://community.tableau.com/s/question/0D54T00000C66CtSAJ/connect-tableau-to-r-and-using-r-scripts-in-tableau

Demikian dan terima kasih.

**Comment by AnthonySSSSS** [comment]
Thank You

Pada tanggal Jum, 4 Nov 2022 20:36, kevbow ***@***.***>
menulis:
